zcash_client_sqlite/src/wallet/init/migrations/add_utxo_account.rs

```diff
@@ -123,3 +123,87 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
         panic!("Cannot revert this migration.");
     }
 }
+
+#[cfg(feature = "transparent-inputs")]
+fn get_transparent_receivers<P: consensus::Parameters>(
```

Comment:
Thanks for copying in the original implementations of these, I was worried about that.